framework/include/mesh/MooseMesh.h

```diff
@@ -740,6 +740,11 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
    */
   std::vector<SubdomainID> getSubdomainIDs(const std::vector<SubdomainName> & subdomain_name) const;
 
+  /**
+   * Get the list of subdomains for the mesh (no active elements included)
+   */
+  std::set<SubdomainID> getSubdomains() const;
```

Comment:
```suggestion
  std::set<SubdomainID> getSubdomainIDsSet() const;
```

Reply:
Is this the same as `meshSubdomains`?

Reply:
It's the member variable from `MooseMesh`, `_mesh_subdomains`

Reply:
Yaqi is right, this routine already exists. It s called meshSubdomains